app/views/navbar.scala.html

```diff
@@ -179,69 +179,64 @@
         });
     }
 
-    // Triggered when 'Sign In' in navbar is clicked
-    // Logs "Click_module=SignIn_location=Navbar_route=</|/audit|/help|...>"
-    $("#sign-in-button").on('click', function(){
+    // The rest of this script logs whichever button is hit in the navbar
+    // Example: Following logs "Click_module=PSLogo_location=Navbar_route=</|/audit|/help|...>"
+    $("#navbar-brand").on('click', function(){
+        logWebpageActivity("Click_module=PSLogo");
+    });
+
+    $("#navbar-sign-in-btn").on('click', function(){
         logWebpageActivity("Click_module=SignIn");
     });
 
```

Comment:
I think you can take out the blank lines between these onclick functions. Just taking up space.

Reply:
Well you can still keep a newline before each new section (the two drop down menu sections)